src/ray/gcs/gcs_server/gcs_placement_group_manager.cc

```diff
@@ -147,6 +148,20 @@ void GcsPlacementGroupManager::RegisterPlacementGroup(
     }
     return;
   }
+  if (!placement_group->GetName().empty()) {
+    auto it = named_placement_groups_.find(placement_group->GetName());
+    if (it == named_placement_groups_.end()) {
+      named_placement_groups_.emplace(placement_group->GetName(),
+                                      placement_group->GetPlacementGroupID());
+    } else {
+      std::stringstream stream;
+      stream << "Placement Group with name '" << placement_group->GetName()
+             << "' already exists.";
```

Comment:
Can you also add a id of a placement group that has the same name? (for users, this will give them better debugging experience).

Reply:
yeah, that's a good idea.